Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM golang:alpine AS build
+
+RUN apk add --no-cache gcc libc-dev git
+
+WORKDIR /src/jx-cdevents-adapter
+
+# Force the go compiler to use modules
+ENV GO111MODULE=on
+ENV BUILDFLAGS=""
+ENV GOPROXY=https://proxy.golang.org
+
+# Copy `go.mod` for definitions and `go.sum` to invalidate the next layer
+# in case of a change in the dependencies
+COPY go.mod go.sum ./
+
+# Download dependencies
+RUN go mod download
+
+ARG debugBuild
+
+# set buildflags for debug build
+RUN if [ ! -z "$debugBuild" ]; then export BUILDFLAGS='-gcflags "all=-N -l"'; fi
+
+# Copy local code to the container image.
+COPY . .
+
+# Build the command inside the container.
+# (You may fetch or manage dependencies here, either manually or with a tool like "godep".)
+RUN GOOS=linux go build -ldflags '-linkmode=external' $BUILDFLAGS -v -o jx-cdevents-adapter
```

Comment:
do we need CGO?

Reply:
Hey @ankitm123 should i put ``` ENV CGO_ENABLED=1``` there?

Reply:
yes, I saw gcc being added in the dockerfile, so I was curious if building the adpater needs CGO library. If it's not, then CGO_ENABLED=0 would be good? Adding C libraries has side effects though 🤔

Reply:
I don't think we need it I copied that Dockerfile from a project I did in the past for us to get started. I added the ENV for GCO_ENABLED. can you please check if its in the right place?

Reply:
And please feel free to optimize the Dockerfile etc if you see room for improvements..